ocean_lib/web3_internal/contract_base.py

```diff
@@ -120,7 +121,7 @@ def get_tx_receipt(tx_hash: str, timeout=20):
                 f"ConnectionClosed error waiting for transaction receipt failed: {e}."
             )
             raise
-        except Exception as e:
```

Comment:
please keep the previous arrangement here and in contract.py. The point is that any exception thrown by the web3 checks, will be caught and raised as VeriftTxFailed.

Reply:
Ok, will modify that right now. So all these web3 checks should be caught as VerifyTxFailed.